src/components/Divider.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+const StyledDiv = styled.div`
```

Comment:
`StyledDivider`. `StyledDiv` is somewhat opaque since `<div />` tags are applicable to a wide range of components.